recipes/inversify-cpp/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class InversifyCppConan(ConanFile):
+    name = "inversify-cpp"
+    description = "C++17 inversion of control and dependency injection container library"
+    topics = ("conan", "inversify-cpp", "ioc", "container", "dependency", "injection")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mosure/inversify-cpp"
+    license = "MIT"
+    settings = "compiler"
+    no_copy_source = True
+    
+    def validate(self):
+        tools.check_min_cppstd(self, "17")
```

Comment:
```suggestion
    @property
    def _compilers_minimum_version(self):
        return {
            "gcc": "9",
            "Visual Studio": "15.7",
            "clang": "7",
            "apple-clang": "11",
        }

    def validate(self):
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, 17)
        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
        if minimum_version:
            if tools.Version(self.settings.compiler.version) < minimum_version:
                raise ConanInvalidConfiguration("{} requires C++17, which your compiler does not support.".format(self.name))
        else:
            self.output.warn("{} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(self.name))
```

Need to validate compiler versions